File: Deberes/1.if-else.py
# Indice de Masa Corporal
peso = float(input("Inserte su peso: "))
estatura = float(input("Inserte su estatura: "))
IMC = peso/(estatura**2) #Formula del IMC 
if IMC < 18.5:
    print("Tienes un peso menor a lo normal")
if IMC >= 18.5 and IMC <= 24.9 : 
    print("Tienes un peso normal")
if IMC > 24.9 and IMC <= 29.9 :
    print("Tienes un peso superior a lo normal")
if IMC > 29.9 :
    print("Tienes obesidad")       

File: Deberes/1.if-else.ts
const peso = Number(prompt("Inserte su peso en kg"));
const estatura = Number(prompt("Inserte su estatura en metros"));
const IMC = peso/(estatura*estatura)
if (IMC < 18.5){
    console.log("Tienes un peso inferior a lo normal")
}
if (IMC >= 18.5 && IMC <= 24.9){
    console.log("Tienes un peso normal")
}
if (IMC > 24.9 && IMC <= 29.9){
    console.log("Tienes un peso superior a lo normal")
}
if (IMC > 29.9){
    console.log("Tienes obesidad")
}